src/main/java/com/kainos/ea/services/RoleService.java

```diff
@@ -19,4 +19,11 @@ public List<RoleResponse> getAllJobRoles()
             throws SQLException, FailedToRetrieveException {
         return roleDao.getAllJobRoles();
```

Comment:
Is this function still used? And if so, which endpoint?

Reply:
changed